designs/virtualization/4.x/snapshots/snapshots.md

```diff
@@ -0,0 +1,85 @@
+---
+parent: Virtualization
+version: 4.x
+---
+
+# VM snapshots
+
+This item will cover updates to vm-snapshots
+- Creating a VM snapshot
+- Snapshots on the VM details page
+- Virtualization snapshot list
+- Restoring a VM from snapshot
+
+## Creating a snapshot
+
+![Virtualization - VMs list](img/0-0.jpg)
+From the virtual machines menu item, the user selects “Create Snapshot” from a VM item’s kebab dropdown.
+The same action can be taken from a VM details Action dropdown.
+
+![Virtualization - turn off VM notification](img/0-1.jpg)
+In case the VM is running, the user will be asked to turn it off to make a snapshot
+
+![Virtualization - Create VM snapshot](img/1-0.jpg)
```

Comment:
Do you think we need the description for the VM here?

Reply:
I'm not sure. I thought that it is not required since the user had selected a VM just want to make sure that while on the Create screen, it is the VM he had chosen. But I'm not against adding a description. WDYT?

Reply:
ya I'm actually saying do we need to even show the description at all. I'm wondering how useful it is for a snapshot.

Reply:
I think it comes down to the question if we think the user will ever need to double-check which item she is doing that (full page form) action on, not just snapshots.

I agree that it is very unlikely to actually take an action on an item and then forget what that item was.
BUT it is a full-page and we need to mention what is the subject of the page.

So this leads me to the question, is the name of the subject (workload) is enough to make it recognizable?

I think that we do need to add the name just below the title, for smaller screens.
I don't see how the right side column does any damage. It is a nice touch.
Having said that, it is a dev effort and we do need to make it a design pattern, so it is hard for me to evaluate the benefit vs. the effort.

Reply:
This discussion will be pushed to a future design. For now, the current implementation was using a modal so no "Creation" page is included.